running_gs_app.md

```diff
@@ -81,14 +94,23 @@ the Simulator section of the
 Coming soon!!!
 
 ### Android
-
-    you@machine:~ $ ./$ANDROID_PATH/scripts/gs_manager.sh start
-
+```shell
+  # Once you use 'hard_stop' to close the gs_manager, you will have to use 'restart' afterwards
+  # instead of 'start' if you want to use the gs_manger again. And so, for convenience,
+  # it's easier to use 'restart'. 
+  $ANDROID_PATH/scripts/gs_manager.sh restart
+```
 ### 4. Guest Science Commanding
+```shell
+  cd $SOURCE_PATH/tools/gds_helper/src
 
-    you@machine:~ $ cd $SOURCE_PATH/tools/gds_helper/
-    you@machine:gds_helper $ python gds_simulator.py
+  # There was an issue found with the program due to some changes from Python 2 to 3
+  # which has been fixed, but has not be updated to the main yet. To avoid the error
+  # run the command below to update the program before you launch it.
+  git checkout develop
```

Comment:
I would prefer if this is a comment since other users may not want to use develop, and switching branches will likely change more stuff. I would suggest to add to the comment saying they should try to get the latest one if they find this issue. Also, I believe git has a way to checkout only certain files to a specific branch, you may want to look into that. Let me know what you think

Reply:
I didn't realize that's what checkout did, my bad.Just wanted to let you know though when I tried checkout only the file it didn't work for me, but I did figure out a way to get it to work and since I couldn't get it to work the way they suggested I'm putting what worked for me.

what they suggested that didn't work for me:
checkout develop -- gds_simulator.py

what worked for me:
checkout develop
checkout develop -- gds_simulator.py

Reply:
Interesting, `git checkout develop -- tools/gds_helper/src/gds_simulator.py` should work, assuming that the working tree is clear (the file was not previously changed) and the user is on the main branch. However, this is just a temporary requirement, so I would suggest leaving the details vague for now, since the requirements for every user may be different.

In summary, I would suggest leaving only the `git checkout develop --  gds..` command as part of the note as a reference and not as an actual step, like "Depending on your environment, you may use the following command... `actual_command`...".  This is to prevent people from actually running the command by mistake.

Let me know what you think. Thanks!

Reply:
Thanks!